js/apps/filter-page/constants.js

```diff
@@ -7,8 +7,8 @@
 			locations: '/dist/js/apps/filter-page/templates/card-locations.html'
 		},
 		urls: {
-			// databases: 'http://ba224964:3100/api/structured-content/databases',
-			databases: 'http://testservices.bcpl.info/api/structured-content/databases',
+			databases: 'http://ba224964:3100/api/structured-content/databases',
```

Comment:
Would you point databases back to testservices? (just need to switch which line's commented out)